src/fu-util-common.c

```diff
@@ -824,10 +824,6 @@ fu_util_convert_description (const gchar *xml, GError **error)
 		g_set_object (&n, n2);
 	}
 
-	/* remove extra newline */
-	if (str->len > 0)
-		g_string_truncate (str, str->len - 1);
-
```

Comment:
doesn't this break `get-devices`?

Reply:
I don't think so- they all use the same printing stuff now, but I'll double check.

Reply:
I don't see any regressions in get-devices.